src/core/ee/emotion.cpp

```diff
@@ -929,3 +949,13 @@ void EmotionEngine::cop2_special(EmotionEngine &cpu, uint32_t instruction)
 {
     EmotionInterpreter::cop2_special(cpu, *vu0, instruction);
 }
+
+Cop0* EmotionEngine::get_cop0()
```

Comment:
Recommend marking function as `const`.

```suggestion
Cop0* EmotionEngine::get_cop0() const
```